Languages/Java/questions.json

```diff
@@ -65,12 +65,40 @@
     },
     {
       "id": 7,
-      "question":"Java programming is not statically-typed, means all variables need not first be declared before they can be used.",
+      "question": "Java programming is not statically-typed, means all variables need not first be declared before they can be used.",
       "details": "Every variable in java needs to be declared first before use, else it throws compilation error",
       "questionType": "bool",
       "trueOrFalse": false,
       "shortAns": "No",
       "tag": "Java"
+    },
+
+    {
+      "id": 8,
+      "question": " When an array is passed to a method, what does the method receive?",
+      "details": "When an array is passed to a method, a reference of the array is received by the method",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": "The reference of the array",
+      "tag": "Java"
+    },
+    {
+      "id": 9,
+      "question": "compareTo() returns",
+      "details": "compareTo() returns an int value",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": " an int value",
+      "tag": "Java"
+    },
+    {
+      "id": 10,
+      "question": " Where does the system stores parameters and local variables whenever a method is invoked?",
+      "details": "The system stores parameters and local vaiables in stack in java",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": "Stack",
```

Comment:
Please make your short answer longer and follow correct syntax.

Reply:
ok sir

Reply:
> Please make your short answer longer and follow correct syntax.

 Sir i have created changes as u have guided please view them..
Thank-You..

Reply:
yes sir i will remember that Thank-you so much..
i learned alot contributing for you